pyproject.toml

```diff
@@ -2,6 +2,57 @@
 requires = ["setuptools", "versioneer[toml]==0.29"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "pymoca"
+dynamic = ["version"]
+license = {file = "LICENSE"}
+description = "A python/modelica based simulation environment."
+keywords = ["modelica", "simulation", "compiler"]
+readme = "README.md"
+authors = [
+    { name = "Pymoca Contributors" },
+]
+maintainers = [
+    { name = "Jack Vreeken", email = "jack@vreeken.me" },
+    { name = "Kent Rutan", email = "gs1150e@icloud.com" },
+]
+requires-python = ">=3.8"
+dependencies = [
+    "numpy >= 1.8.2",
+    "antlr4-python3-runtime == 4.13.*",
+]
+classifiers = [
+    "Development Status :: 4 - Beta",
```

Comment:
Jumping straight from 1-Planning to 4-Beta in one fell swoop! 😃 I'm thinking more Alpha than Beta from the perspective of Modelica language compliance and definition of the API for general usage in new areas, but you have a better feel for current use cases. We can maybe put some of our thoughts on this into a roadmap in #297.

Reply:
Fine, I'll make it Alpha ;)